content/html/encyclopedia/semantic/footer.md

```diff
@@ -0,0 +1,73 @@
+---
```

Comment:
Should we change the folder name from `semantic` to `semantic-html` or `semantic-elements`?

Reply:
So `html/glossaries/tags` and `html/glossaries/semantic-elements` ... I'm trying to imagine the most "pleasing" layout of the structure and I can't find the happiest solution.

I would change it to whatever, but maybe after we merge the current semantic pull requests so it can be done in one?

Reply:
I see, the other pull request had semantic-html. My bad.